sensor/kubernetes/listener/resource_event_handler_impl.go

```diff
@@ -106,6 +106,8 @@ func (h *resourceEventHandlerImpl) sendResourceEvent(obj, oldObj interface{}, ac
 	}
 
 	message := h.dispatcher.ProcessEvent(obj, oldObj, action)
+	// TODO Add context here
```

Comment:
Unaddressed TODO for this PR?

Reply:
This should be addressed by `ROX-17157`. I'll add the JIRA reference in the comment